### ISSUE TITLE
[SSR Public Catalog] Rebase/merge main with SSR code  

### ISSUE THREAD

**Issue by jpvajda** [newrelic/instant-observability-website#44]
We are blocked by this lack of functionality of Gatsby Cloud not supporting private environment variables so our design changes will be done in a branch cut from develop, and our SSR changes will eventually need to merge with the design changes once we are unblocked.

### Acceptance Criteria

- [x] Confirm when the env variable feature will be delivered by Gatsby 
- [ ] Add `NERDGRAPH_URL` environment variable to Gatsby Cloud (value should be `https://api.newrelic.com/graphql`)
- [ ] Add `NEW_RELIC_API_KEY` environment variable to Gatsby Cloud (value should be an API key from the Developer Enablement account on prod)
- [ ] Document the New Relic account and API key used for the site, [here](https://docs.google.com/document/d/1uZIOkRh6iv5j3VXZ-tnLgIQBEETjc1guFG6pc3abIrY/edit?usp=sharing)
- [ ] resolve all the joyous merged conflicts 
- [ ] release to production
- [ ] Notify virtuoso and devex we no longer rely on the OILS service 🛢️

**Comment by jpvajda** [comment]
@aswanson-nr Can we close this out as won't do now that we decided to not release SSR? or is there anything we need to consider in this ticket still?

**Comment by aswanson-nr** [comment]
@jpvajda this can totally be closed.
